### DISCUSSION TITLE
alpine-apk: 3.20 openresty-package-repository

### DISCUSSION THREAD

**Discussion by heidricha** [openresty/docker-openresty#255] [Q&A] [answered]
Hello,

is there a plan to release the openresty pkgs for alpine-3.20 soon? There are several security upgrades in alpine-3.20, and it is already available for the other alpine images.

Thanks!

**Comment by neomantra** [comment]
For build-from-upstream like `alpine-apk`,  the `docker-openresty` project only consumes officially published packages from OpenResty.  You would need to ask in those channels.   Otherwise, you can use the built-from-source `alpine` which [supports Alpine 3.20](https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile#L5).

**Comment by heidricha** [comment]
Or... I can just use 3.20 as Alpine version, but v3.18 for the openresty repo path

Not nice, though but image built succesfully.

**Comment by heidricha** [comment]
OK, I see it now....

the alpine-apk image doesn't need to add extra repository since alpine-3.19. openresty-apk is included in the std alpine apk repository!

https://pkgs.alpinelinux.org/packages?name=openresty&branch=v3.20&repo=&arch=&maintainer=

**Comment by neomantra** [comment]
Thanks for pointing that out, I don't know .  We will still build from upstream for these images, but one can of course make their own images.

**Reply by heidricha** [reply]
What's the upstream? Where exactly the openresty package repositories are forged?

**Reply by neomantra** [reply, marked as answer]
* http://openresty.org/en/linux-packages.html
 * https://github.com/openresty/openresty-packaging
